lib/run-executable.js

```diff
@@ -21,17 +21,20 @@ function runExecutable(args, cwd, logLevel) {
 
     const childProcess = spawn(pathToExecutable, args, {
       cwd,
-      stdio: ['inherit', 'inherit', 'pipe']
+      stdio: pipe
     });
 
     childProcess.on('error', (err) => {
       errors += err;
     });
 
+    childProcess.stderr.pipe(process.stderr);
     childProcess.stderr.on('data', (err) => {
       errors += err;
     });
 
+    resolve(childProcess)
```

Comment:
This is a breaking change in the API, as all promises will resolve as soon as the child process is spawned, rather than when it is terminated.

If you want to propose this for a future semver major version, feel free to open a PR then.